src/lib/components/BodyRenderer/Blocks/Tweet/components/twitter-layout/Tweet/TweetSkeleton.svelte

```diff
@@ -3,22 +3,78 @@
 
 	import Skeleton from '../skeleton/index.svelte';
 
-	export let simple = false;
-
 	let theme = getContext('theme');
 </script>
 
-<div
-	class="container mx-0 my-6 rounded-[5px] bg-tw-tweet-bg-color border border-tw-tweet-border {$theme}"
->
-	<div class="pt-5 pb-2.5 px-5">
-		<Skeleton style="height: 2.25rem" />
-		<Skeleton style="height: 7rem; margin: 1.25rem 0" />
-		<Skeleton style="height: 1.25rem" />
-	</div>
-	{#if !simple}
-		<div class="border-t border-tw-tweet-border h-10 px-5 py-2.5 {$theme}">
-			<Skeleton style="height: 1.25rem" />
+<div class="container rounded-[5px] bg-tw-tweet-bg-color border border-tw-tweet-border ${theme}">
+	<Skeleton>
+		<div class="flex justify-between p-4 border-tw-tweet-border">
+			<div class="flex justify-between mt-2">
+				<span class="avatar animation animate mr-2.5 exclude ${theme}" />
+				<span
+					class="flex flex-col no-underline transition-[color] duration-[0.3s] ease-[ease-in-out] exclude {$theme}"
+				>
+					<span class="author animation animate overflow-hidden ${theme}" title="name" />
+					<span class="username animation animate overflow-hidden ${theme}" title="username" />
+				</span>
+			</div>
+			<img
+				class="icon w-[1.25em] h-[1.25em] object-contain twicon brand exclude"
+				title="View on Twitter"
+				alt="twitter"
+				src={'/twitter.png'}
```

Comment:
```suggestion
				src="/twitter.png"
```
@surajsharma Why not that way?